### ISSUE TITLE
.kz not show expiry date

### ISSUE THREAD

**Issue by alvagor** [caarlos0/domain_exporter#256]
HI. domain exporter does not show the expiration date for .kz. As I understand it, **Domain expired** is not processed

whois -h whois.megahost.kz megahost.kz
                                                                                                                                                                                    
```
This Whois Server for the KZ domain zone is maintained by Megahost Kazakhstan LLC (https://megahost.kz).
Сервер информации о доменных именах в зоне KZ. Поддерживается ТОО "Мегахост Казахстан" (0.4463).

Domain Name............: MEGAHOST.KZ

Organization or Person Using Domain Name
Name...................: Megahost Kazakhstan  LLP
Organization Name......: Megahost Kazakhstan, LLP
Street Address.........: Uralsk, prospekt Evraziya, 59/3
City...................: Uralsk
State..................: Zapadno Kazakhstanskaya
Postal Code............: 090000
Country................: KZ

Administrative Contact/Agent
Contact ID.............: MEGAH-40466057
Name...................: Megahost Kazakhstan  LLP
Phone Number...........: +7.7273562797
Fax Number.............: +7.7112961197
Email Address..........: domain@megahost.kz

Nameserver in listed order
Primary server.........: ns1.megahost.kz
IPv4 address...........: 45.136.59.181

Secondary server.......: ns2.megahost.kz
IPv4 address...........: 95.59.26.195

Domain created.........: 2009-07-21 06:14:35 (GMT+0)
Transfer date..........: 2013-10-05 06:30:00 (GMT+0)
Last modified..........: 2023-01-26 17:18:57 (GMT+0)
Domain expired.........: 2025-07-21 06:14:35 (GMT+0)

Domain status..........: clientRenewProhibited

Registar created.......: GFX
Current Registar.......: MEGAHOST
```

**Comment by caarlos0** [comment]
happy to accept a pr for it

**Comment by caarlos0** [comment]
closed by #258
